src/bundle/Resources/public/scss/_main-nav.scss

```diff
@@ -52,10 +52,11 @@
             padding: .625rem 1rem;
             color: $ez-black;
 
-            &.active {
+            &.ez-active {
```

Comment:
Does the class renaming do a thing? Strange without having a broader context.

Reply:
It's conflict with naming. The tab is looking for `.active` to hide the submenu but it finds the `Content structure` menu item because it also has class `.active`.

Reply:
Is it because of Bootstrap scripts?

Reply:
Indeed